src/digitalmarketplace/components/cookie-banner/cookie-banner.js

```diff
@@ -1,6 +1,8 @@
 import { getCookie, setConsentCookie } from '../../helpers/cookie/cookie-functions'
 import InitialiseAnalytics from '../analytics/init'
 
+const cookieSettingsUrl = '/user/cookie-settings'
```

Comment:
I like pulling this out instead of hardcoding, but I wonder if it should be an instance variable or class variable instead of a global variable?

Reply:
In fact, does the cookie banner need to know about whether it is on the cookie settings page? Could the cookie settings JS be entirely responsible for hiding?

Reply:
This is a good point! Yes, the cookie settings page should deal with it. I'll amend this.